.baseline/checkstyle/checkstyle.xml

```diff
@@ -406,7 +406,7 @@
             <property name="switchBlockAsSingleDecisionPoint" value="true"/>
         </module>
         <module name="DesignForExtension"> <!-- Java Coding Guidelines: Design for extension -->
-            <property name="ignoredAnnotations" value="ParameterizedTest, Test, Before, BeforeEach, After, AfterEach, BeforeClass, BeforeAll, AfterClass, AfterAll"/>
+            <property name="ignoredAnnotations" value="ParameterizedTest, Test, Before, BeforeEach, After, AfterEach, BeforeClass, BeforeAll, AfterClass, AfterAll, TestFactory"/>
```

Comment:
FYI you can't modify this file. It's managed by baseline and will just be reverted when baseline upgrades.

https://github.com/palantir/gradle-baseline/pull/2807

See [the README](https://github.com/palantir/gradle-baseline#compalantirbaseline-checkstyle) for how to add custom suppressions.

Reply:
Hmpf, good flag, sorry should have noticed this!

Think it's still reasonable to add this to the template checkstyle.xml if multiple people run into this? But would need to modify the template here:
https://github.com/palantir/gradle-baseline/blob/ff8ac41def94d235721c39639ce0c2ceaf1ccd4f/gradle-baseline-java-config/resources/checkstyle/checkstyle.xml#L427